easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.5.1-intel-2021b.eb

```diff
@@ -0,0 +1,40 @@
+easyblock = 'PythonBundle'
+
+name = 'netcdf4-python'
+version = '1.5.5.1'
```

Comment:
This version is from 2020, it is more interesting to have the most recent v1.5.8 in `intel/2021b`. Could please update this PR?

Reply:
Sloppy of me not checking later versions before making the PR.. However, I see that latest version if `1.5.7` and that it got merged four days ago:
https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/n/netcdf4-python/netcdf4-python-1.5.7-intel-2021b.eb

Should we just close this one down then?

Reply:
Sure, if 1.5.7 works for you then we can close this PR. Thanks for your contribution @rungitta !